### PR TITLE
wip

### PR DESCRIPTION
**Pull request by sean1588**
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/microsoft-for-startups/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Microsoft for Startups Offer
```
Comment (isaac-pulumi) [inline themes/default/content/microsoft-for-startups/_index.md:2]:
```suggestion
title: Microsoft for Startups Founders Hub Offer
```

File: themes/default/layouts/page/microsoft-for-startups.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
+{{ define "hero" }}
+    <header class="page-hero">
+        <div class="dot-background-container">
+            <div class="dot-background"></div>
+        </div>
+        <div class="dot-overlay"></div>
+        <div class="page-hero-title">
+            <h1 class="rainbow-text mb-0">Microsoft for Startups Offer</h1>
```
Comment (isaac-pulumi) [inline themes/default/layouts/page/microsoft-for-startups.html:8]:
```suggestion
            <h1 class="rainbow-text mb-0">Microsoft for Startups Founders Hub Offer</h1>
```

**Review by isaac-pulumi — APPROVED** [review]

File: themes/default/content/microsoft-for-startups/_index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Microsoft for Startups Founders Hub Offer
```
Comment (ericrudder) [inline themes/default/content/microsoft-for-startups/_index.md:2]: This is so awkward (not your fault!)

Maybe ... Microsoft for Startups:  Founders Hub Offer

Comment (isaac-pulumi) [reply themes/default/content/microsoft-for-startups/_index.md:2]: Microsoft specifically requested this change

**Comment by ericrudder** [comment]
microsoft needs an editor

**Pull request merged**